Octokit.sln

```diff
@@ -101,6 +101,7 @@ Global
 		HideSolutionNode = FALSE
 	EndGlobalSection
 	GlobalSection(NestedProjects) = preSolution
+		{01687D54-1D87-4562-A721-C57F1C94052C} = {CEC9D451-6291-4EDF-971A-D398144FBF96}
```

Comment:
This is just the integration test project being nested. I'll undo this, not a blocker.